relayer/crates/starknet-chain-components/src/impls/tx_response.rs

```diff
@@ -41,7 +44,20 @@ where
                 // We may not need this when we transition to a production chain.
                 chain.runtime().sleep(Duration::from_secs(1)).await;
 
-                Ok(Some(TxResponse { receipt, trace }))
+                let tx_response = TxResponse { receipt, trace };
+
+                chain
+                    .log(
+                        &format!(
+                            "[{}] {:?}",
+                            tx_hash.to_fixed_hex_string(),
+                            tx_response.execution_resources(),
+                        ),
+                        &LevelInfo,
+                    )
+                    .await;
```

Comment:
This is to output the gas usage of a transaction for benchmarking right? If so it should be at debug level as this doesn't feel very useful for general purpose relaying from an operator's point of view.